src/utils/filter_keyword_utils.js

```diff
@@ -0,0 +1,71 @@
+// import React from 'react'
```

Comment:
Looks like the importing of react was made obsolete for this given it is a utility, you may just want to delete this comment

Reply:
Furthermore, I would check each utility you made, and determine if the importing of react is necessary or not. I am not 100% either way. Just worth thinking about!

Reply:
Funny enough it would not work unless I had imported React, I am not sure why but maybe it has to do with it being a util? We should definitely revisit this another time though. Thanks for the heads-up, still do not really understand why I had to import it to get it to work.